pkg/resource/scalable_target/testdata/scalable_target/v1alpha1/readone/desired/missing_field.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: applicationautoscaling.services.k8s.aws/v1alpha1
+kind: ScalableTarget
+metadata:
+  name: unit-testing-scalable-target
+spec:
```

Comment:
nit : can we add a comment about which fields are missing ?

Reply:
added.